Volume_Coupling/Volume_Coupling.H

```diff
@@ -0,0 +1,61 @@
+#ifndef Volume_Coupling_H
+#define Volume_Coupling_H
+
+#include "Interface.H"
+
+#include "Volume_Coupling/Generic_volScalarField.H"
+#include "Volume_Coupling/Generic_volVectorField.H"
+
+#include "fvCFD.H"
+
+namespace preciceAdapter
+{
+namespace Volume_Coupling
+{
+
+class Volume_Coupling
+{
+
+protected:
+
+    //- OpenFOAM fvMesh object
+    const Foam::fvMesh& mesh_;
+
+    //- OpenFOAM Time object (we need to access the timestep to compute the pointMotionU)
+    const Foam::Time& runTime_;
+
+    
+    //- Name of the Porosity
+    std::string nameYour_Volume_Field_ = "Your_Volume_Field";
```

Comment:
Do you want to read this name through the dictionary?

Reply:
This was just a placeholder. I have added `Fluid_Velocity' and 'Temperature'